### DISCUSSION TITLE
llm_config - how to set for each model in AutoGen Studio?

### DISCUSSION THREAD

**Discussion by samvanity** [microsoft/autogen#1500] [Q&A]
Hi, I'm using the latest AutoGen and AutoGen Studio. I'm trying to use local LLMs hosted by TextGenWebUi (openai compatible endpoints). The problem with the current AG Studio is that there's no way to set temperatur, min_p, top_k etc from the Models interface. 

Where in the code can I set these things for each local Models I created?

**Comment by sonichi** [comment]
@victordibia fyi
